src/views/AddItem.jsx

```diff
@@ -1,3 +1,65 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase.js';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(7);
+	const [itemAdded, setItemAdded] = useState(false);
+	const [error, setError] = useState(false);
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			await addItem('my test list', {
+				itemName,
+				daysUntilNextPurchase,
+			});
+			setItemAdded(true);
+			setError(false);
+		} catch (e) {
+			setError(true);
+			setItemAdded(false);
+		}
+	};
+
+	return (
+		<>
+			<form onSubmit={handleSubmit}>
+				<label htmlFor="item-name-input">Item name:</label>
+				<input
+					type="text"
+					id="item-name-input"
+					onChange={(e) => setItemName(e.target.value)}
+				/>
+				<fieldset>
+					<legend>How soon will you buy this again?</legend>
+					<input
+						type="radio"
+						id="soon"
+						onChange={() => setDaysUntilNextPurchase(7)}
```

Comment:
love seeing useState and OnChange being used to update the values of user's choice of soon/not soon/kind of soon. really well done!